console/readdvd/dvd_udf.cpp

```diff
@@ -1126,76 +1121,73 @@ static int UDFGetDescriptor( udf_t *udf, int id,
   int desc_found = 0;
   /* Find Anchor */
   lastsector = 0;
-  lbnum = 256;   /* Try #1, prime anchor */
+  lbnum = 256; /* Try #1, prime anchor */
   terminate = 0;
-  if(bufsize < DVD_VIDEO_LB_LEN) {
+  if (bufsize < DVD_VIDEO_LB_LEN) {
     return 0;
   }
-  
-  if(!UDFGetAVDP(udf, &avdp)) {
+
+  if (!UDFGetAVDP(udf, &avdp)) {
     return 0;
   }
 
   /* Main volume descriptor */
   MVDS_location = avdp.mvds.location;
   MVDS_length = avdp.mvds.length;
-  
+
   i = 1;
   do {
     /* Find  Descriptor */
     lbnum = MVDS_location;
     do {
-      
-      //if( DVDReadLBUDF( device, lbnum++, 1, descriptor, 0 ) <= 0 ) {
-      if( read( udf->dev, descriptor, lbnum++, 1, 0 ) ) {
+
+      // if( DVDReadLBUDF( device, lbnum++, 1, descriptor, 0 ) <= 0 ) {
+      if (read(udf->dev, descriptor, lbnum++, 1, 0)) {
```

Comment:
Codacy found an issue: [Check buffer boundaries if used in a loop including recursive loops (CWE-120, CWE-20).  ](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6072101)